packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -456,6 +469,7 @@ class OnsetPicker extends React.Component {
       <div id={this.props.id} {...customProps}>
         <FieldSet className={cx('fieldset')}>
           {/* Precision */}
+          <legend className={cx(['legend', { 'legend-hidden': isLegendHidden }])}>{legend}</legend>
```

Comment:
I think that we should just pass through `legend` and `legendAttrs` props to the `FieldSet`. If we want an `isLegendHidden` prop it should be added to `FieldSet` and just passed through here for consistency.

Reply:
Sorry, I didn't get that assuming and replying something might create more confusion to the context you're expecting so can you please elaborate that little.

Reply:
+1 to @jmsv6d's comment. The `terra-form-fieldset` component already [does the work](https://github.com/cerner/terra-core/blob/725f3a1b80773dbc8da3353f4acc1a0b518c82a5/packages/terra-form-fieldset/src/Fieldset.jsx#L54-L60) for you, I'd simply pass the prop down.

Reply:
I think I might hold this PR until code changes in Terra-core for terra-form-fieldset gets merged to pass isLegendHidden Prop through onset picker.